xclim/data/fr.json

```
@@ -583,19 +582,19 @@
   },
   "RB_FLASHINESS_INDEX": {
     "long_name": "Indice Richards-Baker de torrentialité",
-    "description": "Indice Richards-Baker mesurant le caractère torrentiel d'un débit de rivière.",
+    "description": "{freq} de R-B indice, un indice mesurant le caractère torrentiel d'un débit de rivière.",
     "title": "Indice Richards-Baker de torrentialité",
     "abstract": "Mesure des oscillations du débit relatif au débit moyen, quantifiant la fréquence et la rapidité des changements de débits."
   },
   "FREQ_ANALYSIS": {
-    "long_name": "Période de retour {mode} {indexer} du flux de {window} jours",
-    "description": "Analyse de fréquence pour le {mode} {indexer} du flux de {window} jours, estimé avec la distribution {dist:f}.",
+    "long_name": "Période de retour du montant de débit",
+    "description": "Analyse de fréquence de débit du courant pour le {mode} {indexer} du flux de {window} jours, estimé avec la distribution {dist:f}.",
```

```suggestion
    "description": "Analyse fréquentielle de type {dist:f} des débits moyens sur {window} jours, pour le {mode} {indexer}.",
```
Vraiment pas certain de ma proposition, mais la version actuelle ne fonctionne pas du tout.